src/huggingface_hub/utils/logging.py

```diff
@@ -88,53 +102,75 @@ def get_logger(name: Optional[str] = None) -> logging.Logger:
 
 def get_verbosity() -> int:
     """Return the current level for the HuggingFace Hub's root logger.
+
     Returns:
-        Logging level, e.g., ``huggingface_hub.logging.DEBUG`` and ``huggingface_hub.logging.INFO``.
-    .. note::
-        HuggingFace Hub has following logging levels:
-        - ``huggingface_hub.logging.CRITICAL``, ``huggingface_hub.logging.FATAL``
-        - ``huggingface_hub.logging.ERROR``
-        - ``huggingface_hub.logging.WARNING``, ``huggingface_hub.logging.WARN``
-        - ``huggingface_hub.logging.INFO``
-        - ``huggingface_hub.logging.DEBUG``
+        Logging level, e.g., `huggingface_hub.logging.DEBUG` and `huggingface_hub.logging.INFO`.
+
+    <Tip>
+
+    HuggingFace Hub has following logging levels:
+
+    - `huggingface_hub.logging.CRITICAL`, `huggingface_hub.logging.FATAL`
+    - `huggingface_hub.logging.ERROR`
+    - `huggingface_hub.logging.WARNING`, `huggingface_hub.logging.WARN`
+    - `huggingface_hub.logging.INFO`
+    - `huggingface_hub.logging.DEBUG`
+
+    </Tip>
     """
     return _get_library_root_logger().getEffectiveLevel()
 
 
 def set_verbosity(verbosity: int) -> None:
-    """Set the level for the HuggingFace Hub's root logger.
+    """
+    Sets the level for the HuggingFace Hub's root logger.
+
     Args:
-        verbosity:
-            Logging level, e.g., ``huggingface_hub.logging.DEBUG`` and ``huggingface_hub.logging.INFO``.
+        verbosity (`int`):
+            Logging level, e.g., `huggingface_hub.logging.DEBUG` and `huggingface_hub.logging.INFO`.
     """
     _get_library_root_logger().setLevel(verbosity)
 
 
 def set_verbosity_info():
+    """
+    Sets the verbosity to `logging.INFO`.
```

Comment:
missing `Returns` sections around here.

Reply:
The `set` don't return anything, so I haven't added returns for all methods returning nothing (or `None`). Would you find it clearer if it was specified?

Reply:
I see, in that case I think these functions shouldn't be `return`ing the output of `set_verbosity` which is `None` (doesn't have to be fixed in this PR, you could open another good first issue for this one).

Reply:
Very good point